### PR TITLE
add v530 benchmark documents

### LINKED ISSUE #7160: 能否补充下测试用的HAProxy的机器规格
## Change Request

**This repository is ONLY used to solve problems related to DOCS-CN.
For other issues (TiDB, TiKV, etc), please move to [other repositories](https://github.com/pingcap/).**

Please answer the following questions before submitting your issue. Thanks!

1. Describe what you find is inappropriate or missing in the existing docs.
在测试方案里面提到了HAProxy，但是在硬件配置里面没有看到HAProxy的规格，能否补充下？

2. Provide some reference materials (documents, websites, etc) if you could.
网页链接[link](https://docs.pingcap.com/zh/tidb/stable/benchmark-sysbench-v5.2.0-vs-v5.1.1)

### PR DESCRIPTION
**Pull request by Yui-Song**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

add v520 benchmark doc and haproxy configurations, close #7160
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by dbsid — APPROVED** [review]

File: benchmark/benchmark-sysbench-v5.3.0-vs-v5.2.2.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+|600|50239.42|51110.04|20.74|20.37|1.73|
+|900|57073.97|57252.74|28.16|27.66|0.31|
+
+v5.3.0 对比 v5.2.2，Update Non-index 性能基本持平，略上升了 0.95%。
+
+![Update Non-index](/media/sysbench_v522vsv530_update_non_index.png)
+
+### Update Index 性能
+
+| Threads   | v5.2.2 TPS | v5.3.0 TPS  | v5.2.2 95% latency (ms) | v5.3.0 95% latency (ms)   | TPS 提升（%）  |
```
Comment (TomShawn) [inline benchmark/benchmark-sysbench-v5.3.0-vs-v5.2.2.md:185]:
```suggestion
| Threads   | v5.2.2 TPS | v5.3.0 TPS  | v5.2.2 95% latency (ms) | v5.3.0 95% latency (ms)   | TPS 提升 (%)  |
```

File: benchmark/benchmark-sysbench-v5.3.0-vs-v5.2.2.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+|600|20998.59|21534.13|46.63|45.79|2.55|
+|900|23420.75|23859.64|64.47|62.19|1.87|
+
+v5.3.0 对比 v5.2.2，Update Index 性能基本持平，略上升了 1.83%。
+
+![Update Index](/media/sysbench_v522vsv530_update_index.png)
+
+### Read Write 性能
+
+| Threads   | v5.2.2 TPS  | v5.3.0 TPS | v5.2.2 95% latency (ms) | v5.3.0 95% latency (ms)   | TPS 提升（%）  |
```
Comment (TomShawn) [inline benchmark/benchmark-sysbench-v5.3.0-vs-v5.2.2.md:197]:
```suggestion
| Threads   | v5.2.2 TPS  | v5.3.0 TPS | v5.2.2 95% latency (ms) | v5.3.0 95% latency (ms)   | TPS 提升 (%)  |
```

File: benchmark/v5.3-performance-benchmarking-with-tpcc.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+   server tidb-2 10.9.39.208:4000 check inter 2000 rise 2 fall 3
+   server tidb-3 10.9.64.166:4000 check inter 2000 rise 2 fall 3
+```
+
+## 测试方案
+
+1. 通过 TiUP 部署 TiDB v5.3.0 和 v5.2.2。
+2. 创建数据库 tpcc：`create database tpcc;`
+3. 通过 tiup bench 导入 TPC-C 5000 Warehouse 数据：`tiup bench tpcc prepare --warehouse 5000 --db tpcc -H 127.0.0.1 -p 4000`。
+4. 运行 `tiup bench tpcc run -U root --db tpcc --host 127.0.0.1 --port 4000 --time 1800s --warehouses 5000 --threads {{thread}}` 命令，通过 HAProxy 向 TiDB 加压，每个并发数各测试 30 分钟。
```
Comment (TomShawn) [inline benchmark/v5.3-performance-benchmarking-with-tpcc.md:116]:
```suggestion
4. 运行 `tiup bench tpcc run -U root --db tpcc --host 127.0.0.1 --port 4000 --time 1800s --warehouses 5000 --threads {{thread}}` 命令，通过 HAProxy 向 TiDB 加压，每个并发数测试 30 分钟。
```

File: benchmark/v5.3-performance-benchmarking-with-tpcc.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+5. 从结果中提取 New Order 的 tpmC 的数据。
+
+## 测试结果
+
+v5.3.0 相比 v5.2.2 在 TPC-C 性能上**略下降了 2.99%**。
+
+| Threads | v5.2.2 tpmC | v5.3.0 tpmC | tpmC 提升（%）|
```
Comment (TomShawn) [inline benchmark/v5.3-performance-benchmarking-with-tpcc.md:123]:
```suggestion
| Threads | v5.2.2 tpmC | v5.3.0 tpmC | tpmC 提升 (%) |
```

**Review by TomShawn — COMMENTED** [review]
minor comments on format

**Review by TomShawn — APPROVED** [review]
LGTM

**Review by shichun-0415 — APPROVED** [review]

**Pull request merged**